src/doc_builder/commands/build.py

```diff
@@ -46,8 +46,9 @@ def build_command(args):
         notebook_dir=args.notebook_dir,
     )
 
+    # dev build should not uodate _versions.yml
     package_doc_path = os.path.join(args.build_dir, args.library_name)
-    if os.path.isfile(os.path.join(package_doc_path, "_versions.yml")):
+    if "dev" not in version and os.path.isfile(os.path.join(package_doc_path, "_versions.yml")):
```

Comment:
Let's see what we use for the "version" when pushing PRs builds from transformers. This assumes there is dev in it, but I was thinking of `pr_xxx` with xxx being the PR number.

Reply:
sounds good to me

Reply:
Ok so it looks like we are going with version being pr_xxx so can you change the dev here to pr?